### DISCUSSION TITLE
Coming over from SyncTrayzor?

### DISCUSSION THREAD

**Discussion by GTR8000** [Martchus/syncthingtray#309] [Q&A]
Does anyone have firsthand experience migrating from SyncTrayzor to Syncthing Tray in Windows? If so, I'd like to hear about your experience and any tips you can offer.

I don't like running abandoned software on my PC, and so if I can find a replacement for SyncTrayzor that is actively maintained, I'd like to do so.

I use the software to sync about a dozen directories between my Android phone (running Syncthing-Fork) with my desktop PC running SyncTrayzor on Windows 11. My setup and needs are fairly basic, with the biggest quirk being that I have all directories on the PC set to "ignore deletes" from the phone. This way I have a safe archive on the PC no matter what I do with files on the phone, and yet anything I delete from the PC will get deleted on the phone.

Is Syncthing Tray going to be able to take the place of SyncTrayzor on my desktop simply by installing the 64-bit (Intel/AMD) Tray application, opening the GUI, and mirroring all of my previous settings?

Thanks for any feedback/suggestions/pointers/tips.

**Comment by tomasz1986** [comment]
> Is Syncthing Tray going to be able to take the place of SyncTrayzor on my desktop simply by installing the 64-bit (Intel/AMD) Tray application, opening the GUI, and mirroring all of my previous settings?

I would say that the main difference between the two is that SyncTrayzor embeds the Web GUI directly, while Syncthing Tray uses its own interface, which is more or less read-only, meaning that you will have to open the proper Web GUI in a Web browser if you want to actually make changes in your Syncthing configuration. Some other features exclusive to SyncTrayzor are its folder picker (which uses the native Windows UI) and file conflict resolver.

How about simply running Syncthing Tray alongside SyncTrayzor and comparing the two directly with your own eyes? You can connect to the same Syncthing instance with both of them simultaneously.

**Comment by Martchus** [comment]
I haven't used SyncTrayzor myself but there's probably no need to "mirror" any settings.

I also recommend you simply test whether you like Syncthing Tray by letting it connect to the Syncthing instance started by SyncTrayzor. If you like to use Syncthing Tray exclusively at some point, make sure to stop SyncTrayzor and especially the Syncthing instance started by it. Then you can start Syncthing via the launcher of Syncthing Tray instead. Checkout the README section about starting Syncthing for details. 

Note that Syncthing itself behaves the same and uses the same configuration regardless how it is started - unless you specify e.g. a custom configuration location. Syncthing Tray will not override any defaults here. If SyncTrayzor doesn't do this as well things should just continue to work without any extra configuration effort regardless which wrapper you use to start Syncthing.

**Comment by GTR8000** [comment]
Ah my mistake, I was under the impression that Syncthing Tray was an all-in-one solution and direct replacement for SyncTrayzor, which I see now that it is not. I guess I'll stick with SyncTrayzor seeing as how it has Syncthing and a GUI all in one, and still works fine for my purposes despite having been abandoned. Thanks.

**Reply by Martchus** [reply]
Syncthing Tray **is** an all-in-one solution as the Windows and generic GNU/Linux builds provided by me bundle Syncthing itself. It is just *possible* to let Syncthing Tray connect to any other Syncthing instsnce as well and to not use the bundled version of Syncthing in favour of using a different one. Again, the README contains details about possible setups.

It is true the Syncthing Tray is not a direct replacement for SyncTrayzor in the sense that it doesn't provide all features SyncTrayzor provides (see answer from @tomasz1986). On the other hand, it probably provides features that SyncTrayzor doesn't provide. So you should probably compare/test both to see which one fits your use cases better.
